pkg/operators/formatters/formatters.go

```diff
@@ -392,6 +393,20 @@ var replacers = []replacer{
 				return nil, fmt.Errorf("port size expected to be 2 bytes")
 			}
 
+			protos := in.GetSubFieldsWithTag("name:proto_raw")
+			if len(protos) != 1 {
+				return nil, fmt.Errorf("expected exactly 1 proto field")
+			}
+			if protos[0].Size() != 2 {
+				return nil, fmt.Errorf("proto size expected to be 2 bytes")
+			}
+
+			protoFieldName := strings.TrimSuffix(protos[0].Name(), "_raw")
+			protoField, err := in.AddSubField(protoFieldName, api.Kind_String)
+			if err != nil {
+				return nil, err
+			}
+
```

Comment:
After #2997 we need to be more conservative. This code should check if there's a proto_raw field, if there is not, it should continue to work as before (perhaps printing a warning), if the proto_raw field is present, it should enrich it. 

(this is important to be able to run gadgets that were compiled with an old version of the headers)